### DISCUSSION TITLE
Is there any way to collapse/expand a group with single click?

### DISCUSSION THREAD

**Discussion by zeroarst** [piroor/treestyletab#3281] [Q&A]
Currently I only see double click, or click that arrow icon on the left to collapse/expand a group.
Is there any way to collapse/expand a group with a single click?

**Comment by irvinm** [comment]
@zeroarst I don't think there is a way to do this directly with TST.  (@piroor would know better if there is a way)

However, it does seem like you could create a helper addon via the TST API to listen for click events and then toggle if the tree is collapsed or expanded:  [Get tree information for specific single tab](https://github.com/piroor/treestyletab/wiki/API-for-other-addons#get-tree-information-for-specific-single-tab), [Listening of notification messages](https://github.com/piroor/treestyletab/wiki/API-for-other-addons#listening-of-notification-messages), [Collapse expanded tree](https://github.com/piroor/treestyletab/wiki/API-for-other-addons#collapse-expanded-tree), [Expand collapsed tree](https://github.com/piroor/treestyletab/wiki/API-for-other-addons#expand-collapsed-tree)
